poh/src/poh_recorder.rs

```diff
@@ -1051,7 +1051,7 @@ pub fn create_test_recorder(
 ) {
     let leader_schedule_cache = match leader_schedule_cache {
         Some(provided_cache) => provided_cache,
-        None => Arc::new(LeaderScheduleCache::new_from_bank(bank)),
+        None => Arc::new(LeaderScheduleCache::new_from_bank(&bank)),
```

Comment:
PR comments won't let me go down further. 
- We clone bank at line 1061
- We clone bank at line 1070
- We access `bank.ticks_per_slot()` at line 1077.

I think we can just extract `ticks_per_slot` and remove a clone.